public/pages/Rules/containers/Rules/components/View/View.tsx

```diff
@@ -0,0 +1,186 @@
+      ${content.queries.map((query: any) => `- ${query.value}`)}
+    `
+  }`;
+
+  return (
+    <>
+      <EuiModalBody>
+        <EuiFlexGroup direction="row" justifyContent="flexEnd">
+          <EuiFlexItem>
+            <EuiFormLabel>Rule Name</EuiFormLabel>
+            {content.title}
+          </EuiFlexItem>
+          <EuiFlexItem>
+            <EuiFormLabel>Log Type</EuiFormLabel>
+            {content.category}
+          </EuiFlexItem>
+        </EuiFlexGroup>
+
+        <EuiSpacer />
+
+        <EuiFormLabel>Description</EuiFormLabel>
+        <div>{content.description}</div>
+
+        <EuiSpacer />
+
+        <EuiFlexGroup direction="row" justifyContent="flexEnd">
+          <EuiFlexItem>
+            <EuiFormLabel>Last Updated</EuiFormLabel>
+            {content.last_updated}
+          </EuiFlexItem>
+          <EuiFlexItem>
+            <EuiFormLabel>Author</EuiFormLabel>
+            {content.author}
+          </EuiFlexItem>
+        </EuiFlexGroup>
+
+        <EuiSpacer />
+
+        <EuiFlexGroup direction="row" justifyContent="flexEnd">
+          <EuiFlexItem>
+            <EuiFormLabel>Source</EuiFormLabel>
+            {content.source}
+          </EuiFlexItem>
+          <EuiFlexItem>
+            <EuiFormLabel>Duplicated from</EuiFormLabel>
+            {/* {content.author} */}
+          </EuiFlexItem>
+        </EuiFlexGroup>
+
+        <EuiSpacer />
+
+        <EuiFlexGroup direction="row" justifyContent="flexEnd">
+          <EuiFlexItem>
+            <EuiFormLabel>Rule level</EuiFormLabel>
+            {content.level}
+          </EuiFlexItem>
+        </EuiFlexGroup>
+
+        <EuiSpacer />
+
+        <EuiFormLabel>Tags</EuiFormLabel>
+
+        <EuiFlexGroup direction="row">
+          {content.tags.map((tag: any, i: number) => (
+            <EuiFlexItem grow={false} key={i}>
+              <EuiBadge color={'#DDD'}>{tag.value}</EuiBadge>
+            </EuiFlexItem>
+          ))}
+        </EuiFlexGroup>
+
+        <EuiSpacer />
+        <EuiSpacer />
+
+        <EuiFormLabel>References</EuiFormLabel>
+        {content.references.map((reference: any, i: number) => (
+          <div key={i}>
+            <EuiLink href={reference.value} target="_blank" key={reference}>
+              {reference.value}
+            </EuiLink>
+            <EuiSpacer />
+          </div>
+        ))}
+
+        <EuiSpacer />
+
+        <EuiFormLabel>False positive cases</EuiFormLabel>
+        <div>
+          {content.falsepositives.map((falsepositive: any, i: number) => (
+            <div key={i}>
+              {falsepositive.value}
+              <EuiSpacer />
+            </div>
+          ))}
+        </div>
+
+        <EuiSpacer />
+
+        <EuiFormLabel>Rule Status</EuiFormLabel>
+        <div>{content.status}</div>
+
+        <EuiSpacer />
+
+        <EuiFormRow
+          label="Detection"
+          fullWidth
+          // helpText={Formikprops.touched.ruleDetection && Formikprops.errors.ruleDetection}
```

Comment:
Not blocking for this draft, but can this comment be removed?

Reply:
refactoring rules code and will remove this in followup PR